stdlib/fun.mli

```diff
@@ -20,19 +20,42 @@
 (** {1:combinators Combinators} *)
 
 external id : 'a -> 'a = "%identity"
-(** [id] is the identity function. For any argument [x], [id x] is [x]. *)
+(** [id] is the identity function. For any argument [x], [id x] is [x].
+
+    example:
+    [List.init 3 id] is [[0; 1; 2]] *)
```

Comment:
I would use the qualified form `List.init 3 Fun.id`, as it would appear in source code in practice. (But maybe this goes again the current practice for other examples ?)

Reply:
I opted for unqualified following the existing convention. I assume it was like this since we're already inside `Fun`. Will take a look at other examples in the docs.